src/pyuvdata/uvbeam/mwa_beam.py

```diff
@@ -265,7 +265,7 @@ def _read_metadata(self, h5filepath):
 
         pol_names = sorted(pol_names)
 
-        dipole_names = np.array(sorted(dipole_names))
+        dipole_names = np.sort(np.asarray(list(dipole_names), dtype=int)).astype(str)
```

Comment:
I believe, this could also have been accomplished as the following. Specifying `key` in the sorted method will call that function on each element, sort based on the output. Up to you if you want to keep the double cast.

```suggestion
        dipole_names = np.asarray(sorted(dipole_names, key=int)))
```

Reply:
Thank you! I thought there was a way to do this without actually converting to ints and back but I couldn't remember what it was.